doc/ref_arch/openstack/chapters/chapter03.rst

```diff
@@ -696,6 +709,8 @@ characteristics of each. Ultimately the decision rests with the operator
 to achieve specific availability target taking into account use case,
 data centre capabilities, economics and risks.
 
+**Topology Overview**
```

Comment:
@pgoyal01 Is Topology Overview a title? It seems not at the right place, there is another Topology Overview below.

Reply:
@karinesevilla That is why I mead this just bolded text